### DISCUSSION TITLE
Build failure on ubuntu 23.10

### DISCUSSION THREAD

**Discussion by MadRoos** [Project-OSRM/osrm-backend#6851] [Q&A]
Hello,

I'm encountering an issue while trying to build OSRM on my machine running Ubuntu 23.10. I followed the instructions in the documentation but ran into difficulties, and the build was unsuccessful.

Steps to Reproduce:

Installed all dependencies as stated in the documentation.
Cloned the OSRM repository.
Ran command: 
cmake .. -DCMAKE_BUILD_TYPE=Release
cmake --build .

Encountered the error (see build_failure.log).
[build_failure.log](https://github.com/Project-OSRM/osrm-backend/files/15131417/build_failure.log)

Expected Behavior:
Expected successful completion of OSRM build without errors.

Actual Behavior:
The build failed.

Ubuntu Version: 23.10

CMake Version: 3.27.4

GCC Version: 13.2.0

Build Logs or Error Output:
[build_failure.log](https://github.com/Project-OSRM/osrm-backend/files/15131417/build_failure.log)

Additional Information:

I've verified that all necessary dependencies are installed.
I also attempted to build on a different version of Ubuntu (e.g., 22.04), and it succeeded.
Any assistance or recommendations on resolving this issue would be greatly appreciated. Thank you!

**Comment by DennisOSRM** [comment]
Looking at your log this seems to be the interesting line:

```
In file included from /home/madroos/install/OSRM/osrm-backend-5.27.1/include/util/coordinate_calculation.hpp:4,
                 from /home/madroos/install/OSRM/osrm-backend-5.27.1/src/util/coordinate.cpp:1:
/home/madroos/install/OSRM/osrm-backend-5.27.1/include/util/coordinate.hpp:66:34: error: ‘int32_t’ is not a member of ‘std’
   66 | using FixedLatitude = Alias<std::int32_t, tag::latitude>;
      |                                  ^~~~~~~
```

And it seems it was fixed in #6632 some time ago. Please try again with latest master branch.
